src/Elastic.CommonSchema.LoggerProvider/build.ps1

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env pwsh
```

Comment:
We can remove this file, will follow up making this project part of the existing build 👍

Reply:
Removed.